src/bucket/Bucket.cpp

```diff
@@ -207,6 +264,9 @@ Bucket::loadKeys(std::set<LedgerKey, LedgerEntryIdCmp>& keys,
             {
                 if (entryOp->type() != DEADENTRY)
                 {
+                    updateReadQuotaForLK(*currKeyIt, entryOp.value());
```

Comment:
I think we might also have to account for read quota if a DEADENTRY is returned. I'd have to double check the fee model, but I think we charge for the size of the key on reads that return null. Again, I'm not sure though and will need to double check.